src/Test/Mock/MockEnvironmentHandler.ts

```diff
@@ -23,20 +23,40 @@ export function MockEnvironmentHandler(environment: FlatEnvironment): Environmen
     return environment;
   }
 
-  function setAllEnvironments(_environments: FlatEnvironment[]): void {
-    return;
-  }
-
   function determineSelected(): FlatEnvironment | undefined {
     throw new Error("Method not implemented.");
   }
 
+  function useIsHalted(): boolean {
+    return environment.halted;
+  }
+
+  function useIsProtectedEnvironment(): boolean {
+    return Boolean(environment.settings.protected_environment);
+  }
+
+  function useIsServerCompileEnabled(): boolean {
+    return Boolean(environment.settings.server_compile);
+  }
+
+  function useIsExpertModeEnabled(): boolean {
+    return Boolean(environment.settings.enable_lsm_expert_mode);
+  }
+
+  function setAllEnvironments(): void {
+    return;
```

Comment:
in case of void functions, you don't need to explicitly return. you can leave the brackets empty.
`  const myMethod = (): void => {}` or  ` function myMethod = (): void => {}` will have the same result.
This comment applies to the MockFeatureManager.ts too

Reply:
That will trigger `Unexpected empty method`